support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -18,7 +18,6 @@ import { trackComponentClick } from 'helpers/tracking/behaviour';
 import { formatAmount } from 'helpers/checkouts';
 import { selectAmount, updateOtherAmount } from '../contributionsLandingActions';
 import { type State } from '../contributionsLandingReducer';
-import ContributionChoicesHeader from './ContributionChoicesHeader';
```

Comment:
can we delete this file now?

Reply:
I nearly did! But it's still being used by the PaymentMethodSelector component

Reply:
Ahhh yeah! Nice 👍